.github/workflows/publish-packages.yml

```diff
@@ -41,14 +41,7 @@ jobs:
           key: pacote-cache-${{ github.run_id }}-${{ github.run_attempt }}
           restore-keys: pacote-cache-
 
-      - uses: actions/checkout@v4
-        with:
-          repository: 'DefinitelyTyped/DefinitelyTyped'
-          path: 'DefinitelyTyped'
```

Comment:
why don't we have to clone? Don't the files have to be on disk somewhere to publish them?

Reply:
Nope; we didn't have to clone before pnpm. When we switched, things broke because we could no longer analyze dependencies through node_modules that didn't exist. Now the files are static and we don't need to use TS anymore so this is legal again.

Reply:
Instead of cloning, `getDefinitelyTyped` streams the DT tarball from GitHub into memory.